### DISCUSSION TITLE
Yderligere sensorer for ugedage

### DISCUSSION THREAD

**Discussion by Rhviid** [scaarup/aula#59] [Ideas]
I forhold til at kunne skrædesy automations vil det være dejligt, hvis man kunne få informationerne underopdelt på dage.

a la løbende 7 dage frem

sensor.vesterbro_skole_barn_monday
sensor.vesterbro_skole_barn_tuesday
etc

Jeg formoder, at data er ustrukturet, så de skal parses på en eller anden måde. 

Et eksempel nedenfor hvad jeg får af data.

tirsdag 20. dec.
Matematik
Per  Jensen

Det er årets sidste matematiktime. 

natur/teknologi
Per  Jensen
Fremlæggelse af projekt.

onsdag 21. dec.
Matematik
Per Jensen

Juleafslutning fra 8-13\.

**Comment by scaarup** [comment]
Er det ugenoter fra Min Uddannelse? Problemet er, som du siger, at data er ustruktureret. Der er ikke noget fast format, så det bliver ikke for sjov at implementere det i integrationen.
Men det kunne da være interessant om du kunne parse data med templating og få en brugbar automation trigger ud af det. Det tror jeg vil være god inspiration for os andre.

**Comment by Rhviid** [comment]
Det er data fra Meebook. Umiddelbart står de faktisk i Meebook struktureret (fordelt på dage og muligvis også fag), så det burde måske kunne lade sig gøre at hive dem ud på dagsbasis. Kodemæssigt er det en del over mit niveau (som er halvstuderet røver på en god dag). Så det er et udviklingsønske fra min side, da jeg ikke selv har evnerne her (uagtet at det vil være rart at kunne).
